src/UI/Wanted/Missing/MissingLayout.js

```diff
@@ -144,6 +148,16 @@ module.exports = Marionette.Layout.extend({
                 }
             ]
         };
+
+        if (this.droneFolder) {
+            leftSideButtons.items.splice(4, 0, {
```

Comment:
I don't like this magically splicing in the button. Options:
- Append Manual Import as well
- Keep as array, and remove the Drone Factory by findIndex.
- Add 'hidden' property and extend ToolbarLayout to skip it if hidden is or evaluates to true.